apps/api/src/dtos/player-search.dto.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { ApiProperty } from "@nestjs/swagger";
+import { IsString, MaxLength } from "class-validator";
+
+export class PlayerSearchDto {
+  @IsString()
+  @MaxLength(16)
```

Comment:
maybe make it only start searching after 3 characters to prevent it matching everything starting with `a` or something like that

Reply:
I had this initially, but I think it seems more responsive if it starts searching from the beginning. Also the list it returns will be the most popular players searched. So I think it will be useful.